src/SUMMARY.md

```diff
@@ -65,6 +65,7 @@
    - [External Applications](./config/external-applications.md)
    - [Printing](./config/print/index.md)
    - [Containers and Virtual Machines](./config/containers-and-vms/index.md)
+      - [chroots](./config/containers-and-vms/chroot.md)
```

Comment:
Needs to be added to list in the index.md file of containers as well.

I think it does mostly fit here, though.

Reply:
it is